### ISSUE TITLE
Lexica (306680)

### ISSUE THREAD

**Issue by codeman101** [ValveSoftware/Proton#4242]
# Compatibility Report
- Name of the game with compatibility issues: Lexica
- Steam AppID of the game: 306680

## System Information
- GPU:  GTX 960
- Driver/LLVM version: 

## Reproduction
1.  try to run game
2. you'll sequential errors popup.
[steam-306680.log](https://github.com/ValveSoftware/Proton/files/5325269/steam-306680.log)

I should add though that unlike the other game I reported recently this game works with GE's latest version but since it doesn't work on the latest official proton version I wanted to report it.

**Comment by kisak-valve** [comment]
Hello @codeman101, `fixme:vulkan:X11DRV_vkCreateWin32SurfaceKHR Application requires child window rendering, which is not implemented yet!` in the log points towards https://bugs.winehq.org/show_bug.cgi?id=45277.

**Comment by codeman101** [comment]
@kisak-valve Oh thanks for pointing that out I saw that in the log as well but didn't realize it was a known bug on wine. Also I wasn't 100% sure that was the culprit.

**Comment by CygnusTerminal** [comment]
> @kisak-valve Oh thanks for pointing that out I saw that in the log as well but didn't realize it was a known bug on wine. Also I wasn't 100% sure that was the culprit.

If you want to run the game, probably Proton-GE or Proton-tkg would help since they include the childwindow patch on their forks.

**Comment by codeman101** [comment]
> If you want to run the game, probably Proton-GE or Proton-tkg would help since they include the childwindow patch on their forks.

yep. I already set my client to use GE for the game I just made this thread since it doesn't work on the official proton not realizing the bug is a documented one. So hopefully after that issue is resolved in the main wine branch this issue can be closed. :)
